src/features/clusters/upsert/ClusterDetails.tsx

```diff
@@ -242,9 +242,9 @@ export function ClusterDetails({
 				/>)
 			}
 
-			{selectedPlan?.planLimits && selectedPlan.resourcesPerInstance && (
-				<ResourcesPerInstance planLimits={selectedPlan.planLimits} resourcesPerInstance={selectedPlan.resourcesPerInstance} />
-			)}
+			{selectedPlan?.planLimits?.totalReadCount! > 0 ? (
```

Comment:
I like it! Alright if I take this PR over? I'm going to move this logic and text into the component because we use the component in a couple places now. And I can fix the lint errors.